extension/src/shared/view/state/imageSchemaForComponent.ts

```diff
@@ -114,5 +124,42 @@ export const makeImageSchemeAtomForComponent = (layers: readonly LayerModel[]) =
           imageIconAtomsAtom: splitAtom(imageIconsAtom),
         } as ImageIconSet;
       }
+      case VALUE_IMAGE_SCHEME: {
+        if (!isValueImageSchemeComponent(component)) return;
+        const imageIcons = component.preset?.defaultValue
+          ? [
+              {
+                id: component.id,
+                value: component.preset.defaultValue,
+                imageUrl: component.preset.defaultValue,
+                // TODO: Use setting value
+                imageColor: "#ffffff", // color: component.preset.imageColor
```

Comment:
TODO: Fix after color is supported in image value field

Reply:
Noted! Thank you!